resources/components/ImageCard.vue

```diff
@@ -193,7 +196,23 @@ module.exports = {
 @import 'mediawiki.mixins';
 @import '../style-variables.less';
 
+/* stylelint-disable selector-class-pattern */
+// Vue automatically adds and removes these classes based on the "name" property
+// we give to the <transition> wrapper component
+.fade-enter-active,
```

Comment:
We should probably prefix transition names just like we do classes, e.g. `wbmad-fade`, so we can avoid the stylelint-disable lines